pkg/util/kube/utils.go

```diff
@@ -135,3 +136,18 @@ func GetVolumeDirectory(pod *corev1api.Pod, volumeName string, pvcLister corev1l
 
 	return pvc.Spec.VolumeName, nil
 }
+
+// IsCRDReady checks a CRD to see if it's ready, so that objects may be created from it.
```

Comment:
I would remove part of the doc `so that objects may be created from it`. If it helps, this could go on the calling side of the function, but not here, since this function could be used for other purposes as well.

Reply:
Why would you remove that part? The whole purpose of its existence is to wait for it to be in a state such that CRs can be created from it.

Reply:
I explained above, `since this function could be used for other purposes as well`. It was a suggestion. I personally prefer to document what the function does but not how or where or why it will be used, when it makes sense.